simpleclient/src/main/java/io/prometheus/client/Summary.java

```diff
@@ -138,22 +138,44 @@ protected Child newChild() {
     return new Child(quantiles, maxAgeSeconds, ageBuckets);
   }
 
+
+  public static class SimpleTimer {
```

Comment:
This would apply to Histograms too. It probably makes the most sense for it to be a top-level class.

It should also have javafoc showing correct usage, and pointing users to the simpler methods they probably actually want.